docs/docs/guide/cli/agent.md

```diff
@@ -0,0 +1,153 @@
+---
+description: DXOS Agent
+label: Agent
+order: 5
+---
+
+# Agents
+
+- Agents are background processes that implement a DXOS network peer.
+- Agents have an identity and represent a user's device.
+- Agents are managed by a daemon process controlled by the `dx agent` CLI commands.
+- By default, agents will automatically sync all Spaces that the user has access to.
+- Agents may be configured to automatically create epochs for Spaces that were created by the associated user.
+
+## Usage
+
+Both the CLI and agents should be configured to use a given profile using either the `--profile` flag or `DX_PROFILE` environment variable.
+The CLI will automatically connect to the agent using the given profile.
+
+To run an agent in the foreground:
+
+```bash
+DX_PROFILE=test dx agent run
```

Comment:
```suggestion
DX_PROFILE=test dx agent start -f
```

Reply:
Move the references to `DX_PROFILE` to later in this guide or into the CLI docs and leave out of this page.